scss/components/input-group.scss

```diff
@@ -74,7 +74,7 @@ $block: #{$fd-namespace}-input-group;
         }
 
 
-        z-index: 2; // Brings focus state above input
+        z-index: map-get($fd-z-index-levels, "first"); // Brings focus state above input
```

Comment:
@ckoutsiaris ~ This one looks like it should be "second"

Reply:
@eboyer This line actually fixes the bug. I am not sure if it breaks something though :( From a quick visual checking, I didn't notice anything.